mne/io/snirf/tests/test_snirf.py

```diff
@@ -545,11 +563,11 @@ def test_sample_rate_jitter(tmp_path):
     lines = "\n".join(line for line in log.getvalue().splitlines() if "jitter" in line)
     assert "Found jitter of 0.9" in lines
 
-    # Add jitter of 1.01%, which is greater than allowed tolerance
+    # Add jitter of 1.02%, which is greater than allowed tolerance
```

Comment:
1.01% no longer triggered the error, possibly because float imprecision meant that the final period was a bit shorter than expected already.